lib/big_query/base/timestamp.ex

```diff
@@ -1,51 +1,57 @@
 defmodule BigQuery.Base.Timestamp do
   import BigQuery.Base.Query
 
-  def timestamp_query(tbl, where_sql, params, interval) do
-    params
-    |> timestamp_params(interval)
-    |> query(timestamp_sql(tbl, interval, where_sql))
+  def timestamp_query(tbl, interval, group_by) do
+    timestamp_params(interval)
+    |> query(timestamp_sql(tbl, interval, group_by))
+    |> group(interval, group_by)
   end
 
-  def timestamp_sql(tbl, interval, where_sql) do
-    """
-    WITH intervals AS (#{timestamp_intervals(tbl, interval, where_sql)})
-    SELECT time, count FROM intervals
-    ORDER BY time ASC
+  def timestamp_sql(tbl, interval, group_by) do
     """
+    WITH intervals AS (#{timestamp_intervals(tbl, interval, group_by)})
+    SELECT time, #{group_by}, count FROM intervals
+    ORDER BY time ASC, #{group_by} ASC
```

Comment:
won't this leave you with `ORDER BY time ASC,  ASC` if group_by ain't there?

Reply:
It should always be there.  There's no longer a `timestamp_sql/2` with a nil 3rd param ... and nothing from timestamp_group chains to this.